src/cbc_sdk/workload/sensor_lifecycle.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Sensor Lifecycle Management for Workloads"""
+
+from cbc_sdk.errors import ApiError
+from cbc_sdk.base import UnrefreshableModel
+
+import logging
+import json
+
+log = logging.getLogger(__name__)
+
+
+class SensorType(UnrefreshableModel):
+    """Represents a sensor type in a request."""
```

Comment:
Not quite clear what is the request. How about this? `Represents a sensor type in a sensor installation request`